src/assets/scripts/client/utilities/validatorUtilities.js

```diff
@@ -0,0 +1,25 @@
+import _isArray from 'lodash/isArray';
+import _isEmpty from 'lodash/isEmpty';
+import _isObject from 'lodash/isObject';
+
+/**
+ * This will return true if it is not an object or it is empty
+ *
+ * @funtion isNotObjectOrIsEmpty
+ * @param input {object}
+ * @return {boolean}
+ */
+export const isEmptyObject = (value) => {
+    return _isObject(value) && _isEmpty(value);
+};
+
+/**
+ * This will return true if it is not an array or it is empty
+ *
+ * @funtion isNotObjectOrIsEmpty
+ * @param input {object}
```

Comment:
this param name should be value  and the type should be listed as `{*}` because is accepts _any_ type.